app/assets/javascripts/admin/addon/components/site-setting.gjs

```diff
@@ -50,13 +53,48 @@ export default class SiteSettingComponent extends Component {
   @service router;
   @service dialog;
   @service siteSettingChangeTracker;
+  @service messageBus;
 
   @tracked isSecret = null;
+  @tracked status = null;
+  @tracked progress = null;
   updateExistingUsers = null;
 
   constructor() {
     super(...arguments);
     this.isSecret = this.setting?.secret;
+
+    const settingName = this.setting.setting;
+
+    if (
+      !!settingName &&
+      (settingName.includes("default_categories") ||
+        settingName.includes("default_tags"))
+    ) {
+      this.messageBus.subscribe(`${settingName}`, this.onMessage);
+    }
+  }
+
+  willDestroy() {
+    super.willDestroy(...arguments);
+    const settingName = this.setting.setting;
+
+    if (
+      !!settingName &&
+      (settingName.includes("default_categories") ||
+        settingName.includes("default_tags"))
+    ) {
+      this.messageBus.subscribe(
```

Comment:
I think this should be unsubscribing instead of subscribing?

Reply:
This hasn't been resolved too as I believe we need to be unsubscribing here and not subscribing.

Reply:
My bad I forgot to mush these changes